lib/history.js

```diff
@@ -0,0 +1,13 @@
+import Ably from "ably";
+
+const rest = new Ably.Rest.Promise({
+  key: process.env.ABLY_SERVER_API_KEY,
+});
+
+const channel = rest.channels.get("headlines");
+
+export async function getHistoricalMessages() {
+  const resultPage = await channel.history({ limit: 5 });
+  const historicalMessages = JSON.parse(JSON.stringify(resultPage.items));
```

Comment:
Why do we need to do this?

Stringify turns into a string and then we parse from a strong back to JSON.

Reply:
Yes, this is a very annoying[issue](https://github.com/vercel/next.js/issues/11993). It seems that you have to re-parse or certain Message object items can't be serialized by props later.

Reply:
Recommend: Add comment about `JSON.parse(JSON.stringify(` and link to the issue.

Reply:
Done in [a65cd39](https://github.com/ably-labs/ably-next-vercel-news/pull/8/commits/a65cd397ed945fe13c86d64b8f36fffaa563f98b)